views/partials/search-form-fields.html

```diff
@@ -24,10 +24,34 @@
   <a href="?advanced_search=true" class="toggle-advanced">Advanced search</a>
 </div>
 <fieldset class="advanced {% if advancedSearchIsVisible %}visible{% endif %}">
-  {{ createSelectMultiple('condition', 'Conditions', 'Disease, disorder, syndrome, illness or injury that is being studied.') }}
-  {{ createSelectMultiple('intervention', 'Interventions', 'Process or action that is the focus of a clinical study. This can include giving participants drugs, medical devices, procedures, vaccines, and other products that are either investigational or already available. Interventions can also include noninvasive approaches such as surveys, education, and interviews.') }}
-  {{ createSelectMultiple('person', 'People') }}
-  {{ createSelectMultiple('organisation', 'Organisations') }}
+  <div class="filter">
+    <label
+        for="condition"
+        class="hastip"
+        title="Disease, disorder, syndrome, illness or injury that is being studied."
+    >Condition</label>
```

Comment:
Since we're here and we're disabling autocomplete, should we also rename this to "Medical Condition" or something more relevant? We're kinda taking away part of its context, so we need to make it more explicit now.